test/test_before_build.py

```diff
@@ -25,8 +25,13 @@
             stored_executable = f.read()
         print('stored_executable', stored_executable)
         print('sys.executable', sys.executable)
+
         # windows/mac are case insensitive
-        assert os.path.realpath(stored_executable).lower() == os.path.realpath(sys.executable).lower()
+        stored_path = os.path.realpath(stored_executable).lower()
+        current_path = os.path.realpath(sys.executable).lower()
+
+        # TODO: This is not valid in an virtual environment
+        # assert stored_path == current_path, '{0} != {1}'.format(stored_path, current_path)
```

Comment:
Whoops, I missed this on the first run through. I guess we can add this test back in, until we're using `build` as the default?

Reply:
We are testing this in build too, so I don't think it can be restored?

Reply:
I don't think so? I don't see a `build_frontend_env` fixture in this file...

Reply:
```suggestion
        # TODO: This is not valid in an virtual environment
        assert stored_path == current_path, '{0} != {1}'.format(stored_path, current_path)
```

Reply:
Let's see, then!